src/options/index.html

```diff
@@ -1,15 +1,24 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>Ghostery Options</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
-  <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
-  <meta content="utf-8" http-equiv="encoding">
-  <link rel="stylesheet" href="/vendor/@whotracksme/ui/src/fonts.css" />
-  <link rel="stylesheet" href="/vendor/@whotracksme/ui/src/reset.css" />
-  <script type="module" src="index.js"></script>
-</head>
-<body>
-  <ghostery-options></ghostery-options>
-</body>
+  <head>
+    <title>Ghostery Options</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
+    />
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type" />
+    <meta content="utf-8" http-equiv="encoding" />
+    <link
+      rel="stylesheet"
+      href="../vendor/@whotracksme/ui/src/styles/fonts.css"
```

Comment:
VS Code can resolve relative paths in html. If we prefer to use absolute paths here too, I can change that.

Reply:
don't have a strong opinion. Please decide as you prefer.